sharepoint/sharepoint-ps/sharepoint-online/Get-SPOApplication.md

```diff
@@ -2,7 +2,7 @@
 external help file: sharepointonline.xml
 Module Name: Microsoft.Online.SharePoint.PowerShell
 online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spoapplication
-applicable: SharePoint Online
+applicable: SharePoint
```

Comment:
>SharePoint [](http://example.com/codeflow?start=12&length=10)

these should be "SharePoint Online" -- here and other places -- I think they were mistakenly updated to just "SharePoint"